### DISCUSSION TITLE
Tested horizon forbidden west (ps4) on kyty

### DISCUSSION THREAD

**Discussion by DuckersMcQuack** [InoriRus/Kyty#70] [General]
Well, i'll say that  will reattempt on every new release of this emu until it finally boots, and see how many versions it took from first attempt till successful boot :P

![NVIDIA_Share_aLRRTRYOW9](https://user-images.githubusercontent.com/9039370/190933695-ff28b448-de96-495b-aadf-3b9f775d1072.png)

**Comment by SoffiMeower** [comment]
Added your submission to the new Compatibility Spreadsheet! If you want, you can continue updating it on here:
https://docs.google.com/spreadsheets/d/1NSf_N57Qffy5CAr2QYpWi0jOMOHJjhXB9JsYNqPqGHk/edit

**Reply by DuckersMcQuack** [reply]
That i will, Thanks :)
